libafl/Cargo.toml

```diff
@@ -67,6 +67,9 @@ tcp_compression = ["tcp_manager", "libafl_bolts/gzip"]
 ## Enable multi-machine support
 multi_machine = ["tokio", "std", "enumflags2", "ahash/std"]
 
+## Dump state of each client on exit
+dump_state = ["std"]
```

Comment:
dump_state should probably always use `libafl_bolts/unsafe_stable_anymap` otherwise what's the point?

Reply:
I don't see where/how this is used?

Reply:
it's not needed i forgot to delete